tests/test_endf.py

```diff
@@ -480,6 +78,28 @@ def test_endftod():
     exp = np.array(exp)
     assert_allclose(obs, exp, rtol = 1e-8)
 
+def test_loadtape():
+    try:
+        assert(os.path.isfile('endftape.100'))
+    except AssertionError:
+        try:
+            import urllib.request as urllib
+        except ImportError:
+            import urllib
+        urllib.urlretrieve("http://www.nndc.bnl.gov/endf/b6.8/tapes/tape.100",
+                    "endftape.100")
+    from hashlib import md5
+    with open("endftape.100", "rb") as f:
+        obs_hash = md5(f.read()).hexdigest()
+    exp_hash = "b56dd0aee38bd006c58181e473232776"
+    try:
+        assert_equal(obs_hash, exp_hash)
+    except AssertionError:
+        raise AssertionError("endftape.100 hash check failed; please try redownloading the endftape.100 data file.")
+    testlib = Library("endftape.100")
+    exp_nuclides = [10010000, 30060000, 40090000, 50110000, 30070000, 60000000, 50100000]
+    obs_nuclides = testlib.structure.keys()
```

Comment:
@scopatz let me try to setup a python3 virtualenv, but i guess your suggestion should be perfect